ansible_ai_connect/ai/api/telemetry/schema2.py

```diff
@@ -82,3 +83,21 @@ class AnalyticsProductFeedback:
     timestamp: str = field(
         default=Factory(lambda self: timezone.now().isoformat(), takes_self=True)
     )
+
+
+@frozen
+class OneClickTrialPlan:
+    created_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    expired_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    is_active: bool = False
+    name: str = field(validator=validators.instance_of(str), converter=str, default="")
+    id: int = 0
+
+
+@frozen
+class OneClickTrialStarted:
+    plans: list[OneClickTrialPlan] = field(factory=list, validator=validators.min_len(1))
+    rh_user_org_id: int = field(validator=validators.instance_of(int), converter=int, default=0)
```

Comment:
Here again the `default=0` is wrong IMO, we would certainly prefer to get an exception if the field is not defined..

Reply:
done